manifests/charts/base/values.yaml

```diff
@@ -19,4 +19,7 @@ base:
 
   # Validation webhook configuration url
   # For example: https://$remotePilotAddress:15017/validate
-  validationURL: ""
\ No newline at end of file
+  validationURL: ""
+
+# Revision is set as 'version' label and part of the resource names when installing multiple control planes.
+revision: ""
```

Comment:
@costinm I think we want to avoid having base be revisioned but I am not sure how we can reconcile that with this issue. Any thoughts?

Reply:
+1, we should not revision base. It is common across revisions.